src/ifixit2zim/utils.py

```diff
@@ -67,15 +76,20 @@ def only_path_of(self, url: str):
         """normalized path part of an url"""
         return self.normalize_ident(urllib.parse.urlparse(url).path)
 
+    @backoff.on_exception(
+        backoff.expo,
+        requests.exceptions.HTTPError,
```

Comment:
Is there a reason to not use `RequestException` so that we retry also on connection timeouts and stuff like that ?